### PR TITLE
Add stop_transceiver/2

### PR DESCRIPTION
**Pull request by mickel8**

### CODE REVIEW

File: lib/ex_webrtc/peer_connection.ex [hunk 1]
```diff
@@ -361,6 +399,17 @@ defmodule ExWebRTC.PeerConnection do
     {:reply, {:ok, transceiver}, state}
   end
 
+  @impl true
+  def handle_call({:add_transceiver, %MediaStreamTrack{} = track, options}, _from, state) do
```
Comment (mickel8) [inline lib/ex_webrtc/peer_connection.ex:403]: Just moved it as I placed set_transceiver_direction in a wronge place earlier

File: lib/ex_webrtc/peer_connection.ex [hunk 2]
```diff
@@ -607,6 +667,123 @@ defmodule ExWebRTC.PeerConnection do
...
+    # * At the end, add remaining transceiver mlines
+    {_, current_local_desc} = state.current_local_desc
+
+    final_mlines =
+      current_local_desc.media
+      |> Stream.with_index()
+      |> Enum.map(fn {local_mline, idx} ->
+        case Enum.find(transceivers, &(&1.mline_idx == idx)) do
+          nil when state.current_remote_desc == nil ->
+            local_mline
```
[the remark this answers is not in the document]
Comment (LVala) [reply lib/ex_webrtc/peer_connection.ex:721]: Also, I would really love to see some comments explaining why these cases might occur.

File: test/ex_webrtc/renegotiation_test.exs [hunk 3]
```diff
@@ -0,0 +1,164 @@
...
+
+  defp continue_negotiation(pc1, pc2) do
+    {:ok, offer} = PeerConnection.create_offer(pc1)
+    :ok = PeerConnection.set_local_description(pc1, offer)
+    :ok = PeerConnection.set_remote_description(pc2, offer)
+    {:ok, answer} = PeerConnection.create_answer(pc2)
+    :ok = PeerConnection.set_local_description(pc2, answer)
+    :ok = PeerConnection.set_remote_description(pc1, answer)
+    :ok
+  end
```
[the remark this answers is not in the document]
Comment (mickel8) [reply test/ex_webrtc/renegotiation_test.exs:163]: Yeah, it should have not have the first create_offer :eyes:

File: lib/ex_webrtc/peer_connection.ex [hunk 4]
```diff
@@ -710,6 +872,23 @@ defmodule ExWebRTC.PeerConnection do
     end
   end
 
+  defp remove_stopped_transceivers(state, :answer, sdp) do
+    # See W3C WebRTC 4.4.1.5-4.7.12
+    transceivers =
+      Enum.reject(state.transceivers, fn
+        %RTPTransceiver{mid: nil} ->
+          false
```
[the remark this answers is not in the document]
Comment (mickel8) [reply lib/ex_webrtc/peer_connection.ex:879]: IMO this is a bug but that's also how the borwser behaves so I left the comment with a link to the issue

**Review by LVala — APPROVED** [review]

**Pull request merged**